enterprise/cmd/frontend/internal/auth/githuboauth/session.go

```diff
@@ -111,11 +111,15 @@ func (s *sessionIssuerHelper) GetOrCreateUser(ctx context.Context, token *oauth2
 	for i, attempt := range attempts {
 		userID, safeErrMsg, err := auth.GetAndSaveUser(ctx, s.db, auth.GetAndSaveUserOp{
 			UserProps: database.NewUser{
-				Username:        login,
+				Username: login,
+
+				// This email is coming from an external source, so we can safely assume
+				// that the email is verified elsewhere.
```

Comment:
nit: I think this is not 100% accurate
```suggestion
				// We always only take verified emails from an external source.
```

see line 64 of this file.